### DISCUSSION TITLE
Deadline Mode

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#495] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/deadline-mode/

**Comment by brunoaugusto** [comment]
![null](<https://i.imgur.com/xW2lTf5.gif>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/deadline-mode/#comment_1174047506>):

> [![null](<https://camo.githubusercontent.com/b6a6b1e3a151966b0b96882269a6c7b84f66f85a14d02b65c21f302042dc130b/68747470733a2f2f692e696d6775722e636f6d2f7857326c5466352e676966>)](<https://camo.githubusercontent.com/b6a6b1e3a151966b0b96882269a6c7b84f66f85a14d02b65c21f302042dc130b/68747470733a2f2f692e696d6775722e636f6d2f7857326c5466352e676966>)

Tá mais pra isso:



![null](<https://i.giphy.com/media/g8GfH3i5F0hby/200.gif>)
